pta-intelligent-search-metadata-extractor/src/main/java/fi/maanmittauslaitos/pta/search/metadata/model/CodeListValue.java

```diff
@@ -0,0 +1,76 @@
+package fi.maanmittauslaitos.pta.search.metadata.model;
+
+import java.util.Objects;
+
+public class CodeListValue {
```

Comment:
Document the parts of the code that are "code list values" but do not use this format. You could list them here

Reply:
Added a list of possible candidates as a comment